source/content/guides/secrets/02-secrets-overview.md

```diff
@@ -70,3 +77,40 @@ Environment overrides provide overrides to a secret value for a specific environ
 4. Environmental overrides have the highest priority. If the override exists, it will become the value provided to the calling function.
 
 ![Secrets Relationships](../../../images/guides/secrets/secrets-relationships.png)
+
+
+### The life of a secret
+
+When a given runtime (e.g. Integrated Composer or an environment php runtime) fetches secrets for a given site (and env), the process will be as follows:
+
+- Fetch secrets for site (of the given type and scopes).
+
+- Apply environment overrides (if any) based on the requesting site environment.
+
+- If the site is owned by an organization:
+
+    - Fetch the organization secrets.
+
+    - Apply environment overrides (if any) based on the requesting site environment.
+
+    - Merge the organization secrets with the site secrets (the following example will describe this process in more detail).
+
+### Example Value Resolution
+Given you have an integrated composer site named `my-org-site` which belongs to an organization `my-org`, and you also have another integrated composer site named `my-personal-site` which belongs to your personal Pantheon account.
+
+When Integrated Composer attempts to get secrets for `my-personal-site` it will go like this:
+- Get the secrets of scope `ic` for `my-personal-site`.
+- Apply environment overrides for the current environment.
+- Look at `my-personal-site` owner. In this case, it is NOT an organization so there are no organization secrets to merge.
+- Process the resulting secrets to make them available to Composer.
+
+On the other hand, when Integrated Composer attempts to get secrets for `my-org-site`, it will go like this:
+- Get the secrets of scope `ic` for `my-org-site`.
+- Apply environment overrides for the current environment.
+- Look at the site owner. It determines it is the organization `my-org`.
+- Get the secrets for the organization `my-org` with scope `ic`.
+- Apply the environment overrides to those secrets for the current environment.
+- Merge the resulting organization secrets with the site secrets with the following caveats:
+    - Site secrets take precedence over organization secrets. This means that the value for site-owned secret named `foo` will be used instead of the value for an org-owned secret with the same name `foo`.
+    - Only the secrets for the OWNER organization are being merged. If the site has a Supporting Organization, it will be ignored.
```

Comment:
I like it here, esp for strategic partners since they are likely to be members of both types of organizations. It's mentioned in the "Organization-owned secrets" section earlier on the same page so it makes sense to represent it in the workflow example I think

Reply:
I agree I like having this note here to make things clear